### ISSUE TITLE
Cannot find module '@azure/ms-rest-nodeauth'.

### ISSUE THREAD

**Issue by TechRazor** [Azure/ms-rest-nodeauth#37]
When testing out the new sdk's I get the following error on the @azure/ms-rest-nodeauth module as the package on npm is missing the dist folder.

```typescript
filename.ts:1:33 - error TS2307: Cannot find module '@azure/ms-rest-nodeauth'.
```

Temporary workaround I'm using:

```bash
cd <my-project-directory>
cd './node_modules/\@azure/ms-rest-nodeauth'
npm run build
```

then in my .ts file use this for the import:

```typescript
import * as msRestNodeAuth from './node_modules/@azure/ms-rest-nodeauth/dist/msRestNodeAuth';
```

**Comment by kpajdzik** [comment]
This is fixed in [version 0.8.2](https://www.npmjs.com/package/@azure/ms-rest-nodeauth/v/0.8.2). Apologies for the problem!

**Comment by TechRazor** [comment]
@kpajdzik thanks for the quick fix!
